frontend/javascripts/viewer/geometries/materials/plane_material_factory.ts

```diff
@@ -570,15 +570,6 @@ class PlaneMaterialFactory {
         },
         true,
       ),
-      listenToStoreProperty(
```

Comment:
@MichaelBuessemeyer I didn't add a listener for the layer's bboxes because I don't know when the layer's bboxes could change 🤔 I think this cannot happen right now. Or can you can up with a case (that exists right now) which contradicts my assumption?